server.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express'),
+      app = express(),
+      router = express.Router(),
+      fs = require('fs'),
+      bodyParser = require('body-parser');
+
+let news;
+
+fs.readFile('news.json', (err, data) => {
+    if (err) throw err;
+    news = JSON.parse(data).articles;
+});
+
+router.use((req, res, next) => {
+    res.setHeader('Access-Control-Allow-Origin', null);
+    res.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type');
+    res.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE');
+    next();
+});
+
+router.get('/news', (req, res) => {
+    res.json(news);
+});
+
+router.post('/news', (req, res) => {
+    res.json(news);
+});
```

Comment:
Post method is not created for reading operation

Reply:
Agree, sorry, forgot to remove.